lib/SILOptimizer/SILOptimizer.cpp

```diff
@@ -0,0 +1,11 @@
+//===--- SILOptimizer.cpp -------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
```

Comment:
Can you add a file level comment saying not to put stuff in this file and that it is only here so we have a single translation unit in the base library?

Reply:
I almost wonder if it would even make sense to change the name of the file to make this clearer.